fastapi_backend/app/schemas.py

```diff
@@ -31,6 +31,4 @@ class ItemRead(ItemBase):
     id: UUID
     user_id: UUID
 
-    class Config:
-        orm_mode = True
-        from_attributes = True
+    model_config = {"from_attributes": True}
```

Comment:
Why this was changed? Please add comments on your own PRs to explain the reviewer about any non-trivial unrelated changes.

Reply:
It was deprecated, I got the warning and updated it.